src/particles/collisions/ElasticCollisionPerez.H

```diff
@@ -90,7 +94,20 @@ void ElasticCollisionPerez (
         ++i1; if ( i1 == static_cast<int>(I1e) ) { i1 = I1s; }
         ++i2; if ( i2 == static_cast<int>(I2e) ) { i2 = I2s; }
       }
-      n12 = n12 / dV;
+
+    }
+
+    // in normalized units, the weights already represent a density, so it must only be rescaled
+    // to SI units. In SI units, the weights represent the amount of particles, so it must be
+    // divided by the volume
+    if (normalized_units) {
+        n1 = n1 * background_density_SI;
+        n2 = n2 * background_density_SI;
+        n12 = n12 * background_density_SI;
+    } else {
+        n1 = n1 / dV;
+        n2 = n2 / dV;
+        n12 = n12 / dV;
```

Comment:
```suggestion
        n1 /= dV;
        n2 /= dV;
        n12 /= dV;
```
We could even do `n1 *= normalized_units ? background_density_SI : 1._rt/dV;` but that doesn't really matter.